### PR TITLE
#4165 [Manager] Property view's metrics and node name info are not sh…

### PR DESCRIPTION
**Pull request by wu87988622**
…owing up after a Paper element is running

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4165 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Pipeline/Pipeline.js [hunk 1]
```diff
@@ -206,9 +207,11 @@ const Pipeline = React.forwardRef((props, ref) => {
         timer = setInterval(async () => {
           const res = await fetchPipeline(currentPipeline.name);
           if (isPaperApiReady) {
-            paperApiRef.current.updateMetrics(
-              res.data[0].objects.filter(object => object.kind !== KIND.topic),
+            const metrics = res.data[0].objects.filter(
```
Comment (eechih) [inline ohara-manager/client/src/components/Pipeline/Pipeline.js]: We should prevent errors of response, or prevent non-existent data

File: ohara-manager/client/src/context/connector/connectorReducer.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ const reducer = (state, action) => {
         isFetching: false,
         data: map(state.data, connector =>
           isKeyEqual(connector, action.payload)
-            ? { ...connector, ...action.payload }
+            ? { ...omit(connector, ['state']), ...action.payload }
```
Comment (eechih) [inline ohara-manager/client/src/context/connector/connectorReducer.js]: 👍

File: ohara-manager/client/src/components/Pipeline/Pipeline.js (hunk 1 above)
Comment (wu87988622) [reply ohara-manager/client/src/components/Pipeline/Pipeline.js]: Thanks

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**